### ISSUE TITLE
Breaks when py is not installed

### ISSUE THREAD

**Issue by JelleZijlstra** [Teemu/pytest-sugar#241]
pytest 7.2.0, released just now, makes `py` an optional dependency (https://docs.pytest.org/en/stable/changelog.html#pytest-7-2-0-2022-10-23), but `pytest-sugar` still relies on it, leading to a crash if `py` is not installed:

```
INTERNALERROR> Traceback (most recent call last):
INTERNALERROR>   File "/home/jelle/ans/venv3.9/lib/python3.9/site-packages/_pytest/main.py", line 268, in wrap_session
INTERNALERROR>     config.hook.pytest_sessionstart(session=session)
INTERNALERROR>   File "/home/jelle/ans/venv3.9/lib/python3.9/site-packages/pluggy/hooks.py", line 286, in __call__
INTERNALERROR>     return self._hookexec(self, self.get_hookimpls(), kwargs)
INTERNALERROR>   File "/home/jelle/ans/venv3.9/lib/python3.9/site-packages/pluggy/manager.py", line 93, in _hookexec
INTERNALERROR>     return self._inner_hookexec(hook, methods, kwargs)
INTERNALERROR>   File "/home/jelle/ans/venv3.9/lib/python3.9/site-packages/pluggy/manager.py", line 84, in <lambda>
INTERNALERROR>     self._inner_hookexec = lambda hook, methods, kwargs: hook.multicall(
INTERNALERROR>   File "/home/jelle/ans/venv3.9/lib/python3.9/site-packages/pluggy/callers.py", line 208, in _multicall
INTERNALERROR>     return outcome.get_result()
INTERNALERROR>   File "/home/jelle/ans/venv3.9/lib/python3.9/site-packages/pluggy/callers.py", line 80, in get_result
INTERNALERROR>     raise ex[1].with_traceback(ex[2])
INTERNALERROR>   File "/home/jelle/ans/venv3.9/lib/python3.9/site-packages/pluggy/callers.py", line 187, in _multicall
INTERNALERROR>     res = hook_impl.function(*args)
INTERNALERROR>   File "/home/jelle/ans/venv3.9/lib/python3.9/site-packages/pytest_sugar.py", line 242, in pytest_sessionstart
INTERNALERROR>     self._sessionstarttime = py.std.time.time()
INTERNALERROR> AttributeError: module 'py' has no attribute 'std'
```
(Invoking `pytest` in a env with `pytest-sugar` but not `py` installed.)

**Comment by foxx** [comment]
I'm getting the same error, and the problem indeed goes away when installing `py`.

**Comment by alexcjohnson** [comment]
The fix is pretty simple - @Teemu does #242 look OK?

**Comment by ygworldr** [comment]
> The fix is pretty simple - @Teemu does #242 look OK?

The root issue is the [ReDoS in py library when used with subversion](https://github.com/advisories/GHSA-w596-4wvx-j9j6) in `1.11.0`. There are no patched versions as yet. Simply adding that version will cause things like dependabot to whine about broken dependencies.

**Comment by ygworldr** [comment]
> The fix is pretty simple - @Teemu does #242 look OK?

This looks good to me. My apologies for the earlier confusion!

**Comment by thewchan** [comment]
Will a release on PyPI be made soon to push this fix?

**Comment by ygworldr** [comment]
> Will a release on PyPI be made soon to push this fix?

Adding `py` as a dependency does not solve the problem of the CVE. Whatever you think about that one, it is a problem… ¯\\\_(ツ)\_/¯

The correct fix (by @alexcjohnson in [242](https://github.com/Teemu/pytest-sugar/pull/242)) still needs to be merged…

**Comment by thewchan** [comment]
Got it, no worries we can wait just wondering

**Comment by thewchan** [comment]
@ygworldr with the fix merged, can we expect a release on PyPI soon?

**Comment by ygworldr** [comment]
> @ygworldr with the fix merged, can we expect a release on PyPI soon?

Why are you asking me? I have nothing to do with `pytest-sugar`. I am just a happy user.

**Comment by thewchan** [comment]
Sorry you replied earlier so I assumed you might have been one of the maintainer. @Teemu do you have any idea?

**Comment by justinmayer** [comment]
@thewchan: As the maintainer and person who merged the fix, @Teemu is presumably well-aware that folks can't take advantage of said fix until it has been published to PyPI. So let's not pester @Teemu and instead wait patiently for the package to be published.

**Comment by thewchan** [comment]
Understood, we just have code in production waiting in this and I'm trying to gauge if I should wait or downgrade on my end.

**Comment by ygworldr** [comment]
> Understood, we just have code in production waiting in this and I'm trying to gauge if I should wait or downgrade on my end.

I am in the same boat. I just removed `pytest-sugar` until the fixed version is out. Furthermore, I looked at a [few alternatives](https://github.com/Teemu/pytest-sugar#similar-projects). However, they are (sadly) all far too immature to use at the moment.

**Comment by thewchan** [comment]
@ygworldr Thanks for sharing! Yeah I'm okay for waiting a bit I think, thanks all

**Comment by Teemu** [comment]
Released 0.9.6

**Comment by ygworldr** [comment]
> Released 0.9.6

Thank you! :heart:
